dist/css.js

```diff
@@ -54,4 +70,9 @@ return JSON.stringify({
     document.querySelectorAll(
       'link[rel="stylesheet"][media*="prefers-color-scheme"]',
     ).length > 0,
+
+  externalCssInHead: countExternalCssInHead(),
+  externalCssAfterHead: countExternalCssAfterHead(),
+  inlineCssInHead: countInlineCssInHead(),
+  inlineCssAfterHead: countInlineCssAfterHead(),
```

Comment:
```suggestion
  externalCssInHead: countExternalCssInHead(),
  externalCssAfterHead: countExternalCssInBody(),
  inlineCssInHead: countInlineCssInHead(),
  inlineCssAfterHead: countInlineCssInBody(),
```

Though I'm not sure this is worth a separate function since it's pretty simple. The example above `css_in_js` inlines this.

Reply:
IMO separate functions are generally easier to read and provide better overview than inline, but if you prefer it inline I can change it.

Reply:
I could go either way. Happy to leave as is.